src/pages/edge/lifecycle-for-edge-network/index.md

```diff
@@ -89,8 +103,12 @@ After you complete your configuration, verify that your rule looks like the foll
 1. Select **Save**.
 2. Rebuild your mobile property and deploy it to the correct environment.
 
-## Add the Lifecycle extension to your app
+### Add the Lifecycle extension to your app
+
+Installing Lifecycle for Edge Network is the same as installing Lifecycle for Analytics as there is only a single mobile SDK Lifecycle extension. The Lifecycle APIs for lifecycleStart and lifecyclePause are the same for both Lifecycle workflows. If your application already includes the Lifecycle extension then you are already setup for Lifecycle for Edge Network.
+
+For implementation details, please reference the Lifecycle documentation to learn how to [add Lifecycle to your app](../../home/base/mobile-core/lifecycle/index.md#add-lifecycle-to-your-app) and [register Lifecycle with Mobile Core and add appropriate the Start/Pause calls](../../home/base/mobile-core/lifecycle/index.md#register-lifecycle-with-mobile-core-and-add-appropriate-start-pause-calls).
 
-For implementation details, please reference the Lifecycle documentation to learn how to [add Lifecycle to your app](../../home/base/mobile-core/lifecycle/index.md#add-lifecycle-to-your-app). as well as [register Lifecycle with Mobile Core and add appropriate the Start/Pause calls](../../home/base/mobile-core/lifecycle/index.md#register-lifecycle-with-mobile-core-and-add-appropriate-start-pause-calls).
+### Add Edge Network extension to your app
 
-In addition, use the following steps to [add the Edge Network extension to your app](../edge-network/index.md#add-the-edge-network-extension-to-your-app).
+Use the following steps to [add the Edge Network extension to your app](../edge-network/index.md#add-the-edge-network-extension-to-your-app).
```

Comment:
Should we also add a section that talks about using datastream overrides for lifecycle edge events? and point to this page https://developer.adobe.com/client-sdks/edge/edge-network/tutorials/send-overrides-rules/?